src/app/messages.service.ts

```diff
@@ -35,4 +36,10 @@ export class MessagesService {
         //    like take updates while some condition is true?
     );
   }
+
+  sendMessage(newMessage:CreateMessage,conversationID:number){
+    this.httpClient.post((`http://localhost:8080/api/conversations/${conversationID}/messages`),newMessage)
+    .subscribe(() => 
+      this.getMessages(String(conversationID)))
```

Comment:
This really isn't necessary. The observable will just fire again in 500ms, right?
And you aren't subscribing to getMessages here so that observable will never get fired from this line.

Reply:
done.